.github/workflows/lint.yml

```
@@ -1,16 +1,16 @@
 name: Lint
 
-on:
+on: 
```

```suggestion
on:
```
White space error at EOL.